pkg/omitter/fileomitter.go

```diff
@@ -9,14 +9,8 @@ type filePatternOmitter struct {
 	filePattern string
 }
 
-func (f *filePatternOmitter) Omit(name, path string) (bool, error) {
-	match, err := filepath.Match(f.filePattern, path)
-	// if there is a match or an error return now
-	if match || err != nil {
-		return match, err
-	}
-	// else check for match with file name
-	return filepath.Match(f.filePattern, name)
+func (f *filePatternOmitter) Omit(path string) (bool, error) {
+	return filepath.Match(f.filePattern, path)
```

Comment:
This won't match patterns like `*.log`. That was part of the spec right?

Reply:
schema says: 

> A file glob pattern on file paths relative to the must-gather root. The pattern should be as described in https://pkg.go.dev/path/filepath#Match

https://github.com/openshift/must-gather-clean/blob/main/pkg/schema/schema.json#L208

Reply:
But the example says https://github.com/openshift/must-gather-clean/blob/e03633880b9c93bfbffbd63d3d647fa0369881f8/pkg/schema/schema.json#L95

Reply:
it does, but that matches log files in the must gather root folder, which there aren't any as of today. But agreed, that example is confusing, will rewrite it.

Reply:
This was in the design doc:
```
   # this omits all log files
   - type: file
     pattern: "*.log"
```

Reply:
that's actually a mistake, since it mentions it would use relative path globbing later on. I'll fix the design doc later to clarify.

Reply:
fixed the design doc to make it clear the globbings are fully by path